ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -4052,6 +4053,49 @@ def test_torch_instance_ne(
     )
 
 
+# ne_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="ne_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        min_value=-1e04,
+        max_value=1e04,
+        allow_inf=False,
+    ),
+)
+def test_torch_instance_ne_(
+    dtype_and_x,
+    init_num_positional_args: pf.NumPositionalArgFn,
```

Comment:
Don't worry about the type hints, they are not needed here. Could you remove them please?

Reply:
@ZoeCD done!